src/exec.js

```diff
@@ -19,6 +19,10 @@ common.register('exec', _exec, {
 // Node is single-threaded; callbacks and other internal state changes are done in the
 // event loop).
 function execSync(cmd, opts, pipe) {
+  if (!common.execPath) {
+    common.error('Unable to find a path to the node binary. Please manually set common.execPath');
+  }
```

Comment:
@freitagbr `execAsync` doesn't actually depend on it. It uses `child_process.exec()` directly, instead of our `execSync` which uses something like `child_process.execSync(process.execPath + ' script.js')`. You can do a simple search for `process.execPath` and you'll see it only shows up in `execSync`.